python/tank_vendor/shotgun_authentication/defaults_manager.py

```diff
@@ -18,6 +17,18 @@ class DefaultsManager(object):
     authenticating a user.
     """
 
+    def __init__(self):
+        """
+        Consructor.
+
+        Reads the default host and login from disk.
+        """
+        self._host = session_cache.get_current_host()
```

Comment:
This new behaviour with the session_cache stuff feels really nice. Like a missing piece that now got added.